public/os-files/kernal.js

```
@@ -81,11 +81,123 @@ window.os = {
+    } catch (e) {
+        console.error('Kernel: Error parsing webOS_storageConfig:', e);
+    }
+
+    if (storageConfig && storageConfig.indexedDBDrive && storageConfig.indexedDBDrive.configured && storageConfig.indexedDBDrive.mounted) {
+        const idbDriveConfig = storageConfig.indexedDBDrive;
+        // Ensure driveLetter is a single character, as customLetter might be just the letter.
+        const driveLetter = (idbDriveConfig.customLetter || idbDriveConfig.driveLetter || 'B').charAt(0);
+        const sizeMB = idbDriveConfig.sizeMB || 256; // Default size if not specified
+        const sizeBytes = sizeMB * 1024 * 1024;
+
+        if (window.FileSystemDrivers && window.FileSystemDrivers.IndexDBRevised) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)